### DISCUSSION TITLE
Screen mode shortcut hasn't listed yet in key configuration

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19193] [Q&A] [answered]
There is no way to change key binding to this so its even unavailable to translate for now (I guess?). Is it intentional?
![image](https://user-images.githubusercontent.com/86934170/179401133-3fe8cdd6-352a-42af-b248-649b423cdb28.png)
![image](https://user-images.githubusercontent.com/86934170/179401169-856e4079-2509-4c27-9135-077147400390.png)

**Comment by peppy** [comment, marked as answer]
You can use alt-enter or ctrl-f11. Not sure this needs to be bindable as it's a generally accepted standard.
